plugins/sentry/src/data/production-api.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SentryIssue } from './sentry-issue';
+import { SentryApi } from './sentry-api';
+
+const API_BASE_URL = 'http://localhost:7000/sentry/api/0/projects/';
```

Comment:
The backend could be running on another port I assume, would it be enough to check if `process.env.PORT` is set and use that if so?

Reply:
@marcuseide Good point! I changed it to use `API_HOST` variable instead 🙂